crate_universe/src/metadata/metadata_annotation.rs

```diff
@@ -395,18 +395,20 @@ impl Annotations {
                         extra
                     })
                     .cloned()
-                    .collect();
+                    .sum();
 
-                if !extras.is_empty() {
+                crate_extra.apply_defaults_from_package_metadata(&pkg.metadata);
```

Comment:
I was thinking that the testing would just be in this file and cover the changes around here. Not necessarily a brand new test target. I think that'd get sufficient coverage of the functionality, right?

Reply:
Ah, I get it now. Good call. I have added that test.